### ISSUE TITLE
Feat: CognitoIdentityPool links

### ISSUE THREAD

**Issue by urosbelov** [sst/ion#510]
Hello to all, yesterday I tried to link **CognitoIdentityPool** to **Nextjs** app, and ION cannot manage to make Resource for it.
I have also tried to link it to Function, but again, without succeed.

Did anyone have same issue?

**Comment by jayair** [comment]
Can you share what you tried and if you got an error for it?

**Comment by urosbelov** [comment]
1. I have created CognitoIdentityPool
`export const identityPool = new sst.aws.CognitoIdentityPool("IdentityPool", {userPools: [{userPool: cognitoUserPool.id,client: cognitoClientWeb.id}]});`

2. Linked it to Nextjs App
`export const web = new sst.aws.Nextjs("TestApp", {link: [cognitoUserPool, cognitoClientWeb, identityPool], path: "packages/frontend", openNextVersion: "3.0.2"});`

And in Nextjs app there is no access to `Resource.IdentityPool.id`. Other links, **cognitoUserPool** and **cognitoClientWeb** works as expected.

Then I tried to link it to lambda function to test, and same behavior.
